Mention Vercel AI SDK in docs

arnaugomez opened the pull request:
Add docs of this PR: https://github.com/ueberdosis/tiptap-pro/pull/300

arnaugomez committed:
feat: mention vercel ai sdk in docs

alexvcasillas reviewed (approved):
Looks good, though isn't there a page on how to use it?

arnaugomez committed:
feat: add example on how to use the ai agent adapter

arnaugomez committed:
feat: provide links to different apis for reference

arnaugomez commented:
> Looks good, though isn't there a page on how to use it?

I've added a more detailed explanation on how it's used, with an example. It was already explained in the Configuration page of the extension, but I've added another example to help the reader understand.
